.github/workflows/changesets.yaml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on:
+  push:
+    branches: [main]
+    paths-ignore:
+      - 'docs/**'
+      - 'templates/**'
+      - 'examples/**'
+      - 'assets/**'
+      - '**.md'
+      - 'LICENSE'
```

Comment:
Not really. If we add a file that would have not been included, then the action would not trigger and we will then miss a release (or a test, in your following comment)
With the proposed code, it will trigger by default if a non-included file is changed, but it can be later fine-tuned in changing this file.